src/Components/Icons/index.css

```diff
@@ -0,0 +1,11 @@
+body {
```

Comment:
Please do not style html elements directly. Also, I think we should add a nice `reset.css` and a `normalize.css`, so that we don't have to do this kind of styling here.

Reply:
Yep! I'll add those files to a Layout Component and apply on the page.